aries_cloudagent/core/conductor.py

```diff
@@ -462,15 +467,29 @@ async def outbound_message_router(
             message: An outbound message to be sent
             inbound: The inbound message that produced this response, if available
         """
+        event_bus = profile.inject(EventBus, required=False)
```

Comment:
Doing required=false followed by an assertion that the event bus exists doesn't make much sense. Should probably just do required = true.

Reply:
is that what that thinggy does? Brilliant!

Reply:
pyright is not happy, but that's beside the point. I have updated the code as requested.

Reply:
Indeed, this is an issue with how the inject method is typed. Not sure we can avoid it without touching a bunch of stuff not relevant to this branch.